alws/constants.py

```diff
@@ -55,13 +56,45 @@ class PermissionTriad:
 
 
 class ReleasePackageTrustness(enum.IntEnum):
+    """
+    Enum representing the trustworthiness of a release package. The trustworthiness is shown in different colors
+    on the web UI. The values correspond to:
+
+    UNKNOWN (0): The trustworthiness of the package is unknown, represented in grey on the UI.
+    MAXIMUM (1): The package has maximum trustworthiness, represented in green on the UI.
+    MEDIUM (2): The package has medium trustworthiness, represented in yellow on the UI.
+    LOWEST (10): The package has the lowest trustworthiness, represented in red on the UI.
+
+    Methods
+    -------
+    decrease(trustness: int) -> int:
+        Decreases the level of trustness by one step from its current value, unless it's already at the lowest.
+    """
+
     UNKNOWN = 0
```

Comment:
I don't get why we need `MEDIUM` to be 2 and `LOWEST` 10. If we don't plan to set the trustness to something between 2 and 10, why not doing a ```UNKNOWN, MAXIMUM, MEDIUM, LOWEST = range(4)``` here?

Reply:
Fair enough, fixed.